RELEASE-NOTES.md

```diff
@@ -114,6 +115,7 @@ Ensure the configured path for storing the blockchain exists:
 ```
 mkdir /tmp/node/my_db
 ```
+As of release "v0.5.0-<placeholder>", as the chain format changed from the previous release, please ensure that you do not reuse a persisted blockchain produced by the previous releases "v0.4.x".
```

Comment:
I see https://www.pivotaltracker.com/story/show/153989778 has in title `v0.5.0`, I think you can use that rather than "v0.5.0-<placeholder>".